components/org.wso2.carbon.config.configuration/src/test/java/org/wso2/carbon/config/configuration/configprovider/ConfigProviderImplTest.java

```diff
@@ -263,24 +235,23 @@ public void xmlFileConfigMapTestCase() throws IOException, SecureVaultException
     }
 
     @Test(expectedExceptions = ConfigurationException.class, description = "This test will test functionality " +
-            "when " +
-            "xml config file not found")
+            "when xml config file not found")
     public void xmlFileNotFoundTestCase() throws ConfigurationException {
         ConfigFileReader fileReader = new XMLBasedConfigFileReader(getFilePath("Example1.xml"));
         ConfigProvider configProvider = new ConfigProviderImpl(fileReader);
-        Configurations configurations = configProvider.getConfigurationObject(Configurations.class);
+        TestConfiguration configurations = configProvider.getConfigurationObject(TestConfiguration.class);
         Assert.assertNull(configurations, "configurations object should be null");
     }
 
 
     @Test(description = "This test will test functionality when using yaml config file")
     public void yamlFileConfigObjectTestCase() throws IOException {
-        Path resourcePath = Paths.get(basedir, "src", "test", "resources", "conf", "Example.yaml");
+        Path resourcePath = configPath.resolve("Example.yaml");
         File file = resourcePath.toFile();
         try (FileInputStream fileInputStream = new FileInputStream(file)) {
             Yaml yaml = new Yaml();
             Map map = yaml.loadAs(fileInputStream, Map.class);
-            Map configurationMap = (Map) map.get("configurations");
+            Map configurationMap = (Map) map.get("testconfiguration");
```

Comment:
Might be better to move "testconfiguration" key to a constant to prevent problems that may occur due to changes of the key value in different locations.